Create local environment template

BronsonQuick opened the pull request:
Fixes #15

BronsonQuick committed:
Remove the static files

BronsonQuick committed:
Add plugins and themes directories

BronsonQuick committed:
Initial pass of the readme

BronsonQuick committed:
Merge in master

ntwb commented on CNAME:
```diff
@@ -1 +0,0 @@
-wpaustralia.org
```
This file needs to remain until we have a host hosting setup for https://wpaustralia.org

ntwb commented on README.md:
```diff
@@ -1,2 +1,43 @@
+
+You can get up and running by doing the following:
+1. Open Terminal.
+2. Change directories into the a folder where you would like to download the project. e.g. `cd /Volumes/Sites`
```
`... directories into the folder`, rather than `... directories into the a folder` please

ntwb commented on README.md:
```diff
@@ -1,2 +1,43 @@
+
+We have bundled a few Chassis Extensions to assist with development. We've included the following extensions:
+* [Xdebug](https://github.com/Chassis/Xdebug) - This extensions automatically sets up the Xdebug php module for you.
+* [Mailhog](https://github.com/Chassis/mailhog) - This extension setup Mailhog which catches all the email WordPress sends. You can see the dashboard for it by visiting. [http://wpaustralia.org:8025/]
```
`http://wpaustralia.local:8025/` rather than `http://wpaustralia.org:8025/` please

ntwb commented on README.md:
```diff
@@ -1,2 +1,43 @@
+
+## Useful commands
+
+* `vagrant up` - Run this in the directory you installed this project in to boot up your development environment. e.g. `cd /Volumes/Sites/wpaustralia; vagrant up` then browse to [http://wpaustralia.local](http://wpaustralia.local)
```
"Run this from the `wpaustralia` directory where you downloaded and installed this project"

ntwb commented on README.md:
```diff
@@ -1,2 +1,43 @@
+
+### Development Process
+
+If you're new to give them here are a few steps to help you get started:
```
???

ntwb reviewed (changes requested):
As per my comment on the `CNAME` file deletion, we can't delete any of these files until we have DNS pointed at out new host (which at the time of writing we do not currently have)

BronsonQuick committed:
Fix english

BronsonQuick committed:
Fix URL

BronsonQuick committed:
Alter the wording

BronsonQuick committed:
Alter the git instructions

BronsonQuick commented:
@ntwb Thanks for the review. Sorry for lots of horrible English as I was rushing and didn't proofread anything 🤣

I've added a new branch called `development` and changed this PR to merge into that. That way we can keep the CNAME stuff for Github in `master` and work on adding themes and plugins in development then when we're ready to switch to a WordPress install on the site we can merge development into master and alter the workflow.

ntwb commented:
Thanks a bunch for the PR and follow up 💐

ntwb reviewed (approved):
LGTM (Looks Good To Me) 👍
